src/backend/src/prisma/schema.prisma

```diff
@@ -1020,3 +1029,26 @@ model Graph_Collection {
   organizationId String
   organization   Organization @relation(fields: [organizationId], references: [organizationId])
 }
+
+model Announcement {
+  announcementId   String       @id @default(uuid())
+  text             String
+  usersReceived    User[]       @relation("receivedAnnouncements")
+  dateMessageSent  DateTime     @default(now())
+  dateDeleted      DateTime?
+  senderName       String
+  slackEventId     String       @unique
+  slackChannelName String
+  organizationId   String
+  organization     Organization @relation(fields: [organizationId], references: [organizationId])
+}
+
+model PopUp {
+  popUpId        String       @id @default(uuid())
+  text           String
+  iconName       String
+  users          User[]       @relation("userPopUps")
```

Comment:
this is usersUnread right? And when they read the popup they get removed from this list?